src/datatypes/character/CharacterSet.hpp

```diff
@@ -107,6 +107,22 @@ namespace spy::character {
              */
             iterator getByUUID(const util::UUID &uuid);
 
+            /**
+             * Searches the set for a character at the given point. Returns first match.
+             * @param  p Point to search for. Has to be in the map.
+             * @return Const iterator to the found character if one at the specified point exits, otherwise a
+             *         const iterator to the end of the set.
+             */
+            [[nodiscard]] const_iterator findByCoordinates(const util::Point &p) const;
```

Comment:
Diese methoden bitte in eine utility klasse

Reply:
mit in GadgetUtils bzw. jetzt GameLogicUtils